Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/View.purs

```diff
@@ -112,14 +115,26 @@ view push state =
       , recentRideView state push
       , headingView state (getString ALL_TOPICS)
       , allTopicsView state push
-      , apiFailureView state push
+      , apiFailureView state push 
       ]
     , linearLayout
       [ width MATCH_PARENT
       , height MATCH_PARENT
       , visibility if state.props.isCallConfirmation then VISIBLE else GONE
       ][PopUpModal.view (push <<< PopupModelActionController) (callConfirmationPopup state)]
-
+    , deleteAccountView state push
+    , linearLayout
+      [ width MATCH_PARENT
+      , height MATCH_PARENT
+      , background Color.lightBlack900
+      , visibility if checkAccountStatus state ST.CONFIRM_REQ then VISIBLE else GONE
+      ][ PopUpModal.view (push <<<  PopUpModalAction) (requestDeletePopUp state ) ]
```

Comment:
Instead of creating two different linear layouts both the pop up modals can be created inside same linear layout.
 use the check added to visibility for calling the functions.